tests/active-active-rhel7-proxy/variables.tf

```diff
@@ -18,12 +18,16 @@ variable "acm_certificate_arn" {
   description = "The ARN of an existing ACM certificate."
 }
 
-variable "license_file" {
-  type        = string
-  description = "The local path to the Terraform Enterprise license to be provided by CI."
-}
-
```

Comment:
We should retain this variable so that the interface for the test modules used by ptfe-replicated remains stable. The test modules can internally adapt between this and the expected input of the secrets fixture.

Reply:
input variable "license_file" is retained and is used to input license file path in fixture module invocation